### DISCUSSION TITLE
How to destroy window in formation studio?

### DISCUSSION THREAD

**Discussion by slavik2011** [ObaraEmmanuel/Formation#23] [Q&A] [answered]
When i try run `app1.destroy()`, program returns error

`AttributeError: 'AppBuilder' object has no attribute 'destroy'`

Can I destroy this window?

**Comment by ObaraEmmanuel** [comment, marked as answer]
@slavik2011 thank you for using Formation. You can access the underlying toplevel widget that contains all these methods as follows

```python
app1._app.destroy()
```

The `_app` attribute is the `tkinter.Tk` object while `app1` is an `AppBuilder` object. The interface is still a work in progress so I will work to make this more obvious in future.
